events/cmd/root.go

```diff
@@ -25,6 +27,9 @@ func EventCmd(cdc *codec.Codec) *cobra.Command {
 		},
 	}
 
+	cmd.Flags().String(flags.FlagNode, "", "The node address")
+	_ = viper.BindPFlag(flags.FlagNode, cmd.Flags().Lookup(flags.FlagNode))
```

Comment:
Curious: Is this FlagNode variable related to the other changes or just a fix while editing things?

Reply:
that ensures flag `node` processed for `events` command